app/models/spree/gateway/komoju_konbini.rb

```diff
@@ -0,0 +1,22 @@
+module Spree
+  class Gateway::KomojuKonbini < KomojuGateway
+    def authorize(money, source, options)
+      details = {
+        type:  "konbini",
+        store: source.convenience,
+        phone: options[:billing_address][:phone]
+      }
+      options = change_options_to_dollar(options) if options[:currency] == "JPY"
```

Comment:
I think no problem. Komoju can captures only JPY.
Spree will receives error response from Komoju, then spree will saves the error log.